### DISCUSSION TITLE
Having issue setting up the demo

### DISCUSSION THREAD

**Discussion by Westernwells** [vendurehq/vendure#1082] [Q&A]
I have tried setting up the demo for days now using PostgreSQL as a database but getting SSL issues.
![image](https://user-images.githubusercontent.com/32157579/132995005-f10c519b-cea8-4b45-b447-888e37c7a28f.png)

**Comment by michaelbromley** [comment]
Hi,

Looks like something related to the config of your pg instance. Maybe try this - create a Vendure project and use SQLite to start off with just to get the scaffold created. Then after that, change the `vendure-config.ts` file to point to your pg database and debug it from there. You might have access to better error reporting that way.

**Reply by justsee** [reply]
Yes `@vendure/create` doesn't configure ssl for the db connection, so it will fail as above if you specify any third-party PostgreSQL host string (to providers like Digital Ocean).

Once it fails it's just a matter of adding the `ssl` configuration to the `vendure-config.ts` as outlined here: https://node-postgres.com/features/ssl.
